bigcommerce/big-design
# feat(docs): fix button group issue (676)

## What?
**Issue1**: When destructive action buttons are disabled (hidden under the ellipsis), the first destructive action button is highlighted, and it should not be highlighted. For more context: (https://github.com/bigcommerce/big-design/issues/676)
**Issue2**: All buttons overlay when rendering just one action button + the ellipsis on the screen..
**Issue3**: The ellipsis appears when all the action buttons fit inside a container.

**Images:**
Issue1:
<img width="659" alt="Screen Shot 2022-02-23 at 5 31 52 PM" src="https://user-images.githubusercontent.com/39739180/155427280-7e3d4077-c18f-4153-9613-f2dc6f8e3101.png">

Issue2:
<img width="316" alt="Screen Shot 2022-02-23 at 5 30 33 PM" src="https://user-images.githubusercontent.com/39739180/155427148-89f251cb-b87f-4f6a-88a4-db43a6e6bfef.png">

Issue3: 
<img width="974" alt="Screen Shot 2022-03-08 at 12 27 25 PM" src="https://user-images.githubusercontent.com/39739180/157316619-429e859f-80f1-4867-9ccf-7ae9346b9b7d.png">

## Why?
To fix bugs.

## Screenshots
**Issue 1 and 2:** 


https://user-images.githubusercontent.com/39739180/155427754-7aaba7c5-766e-42f6-bdff-ddc768f0db41.mp4


**Issue 3:**
<img width="1009" alt="Screen Shot 2022-03-08 at 2 11 22 PM" src="https://user-images.githubusercontent.com/39739180/157317055-55e6e5e2-b3cd-4199-8b11-e2fb1819ef0b.png">



## Testing/Proof
- Run ButtonGroup tests.
- Run Button tests

- Use a `ButtonGroup` component, and add some destructive action buttons. Make sure the `highlight` (color red) doesn't appear when using the `disabled` prop.

```jsx
 <ButtonGroup
      actions={[
        { actionType: 'destructive', text: 'Something destructive 1', disabled: true },
        { actionType: 'destructive', text: 'Something destructive 2', disabled: true },
        { actionType: 'destructive', text: 'Something destructive 3', disabled: true },
        { text: 'View in BigCommerce' },
      ]}
    />
```
